docs/source/sitemap.xml

```diff
@@ -0,0 +1,14 @@
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  <url>
+    <loc>https://docs.kedro.org/en/stable/</loc>
+    <lastmod>2024-08-01T18:53:11.571849+00:00</lastmod>
+    <changefreq>monthly</changefreq>
+    <priority>1.0</priority>
+  </url>
+  <url>
+    <loc>https://docs.kedro.org/en/latest/</loc>
+    <lastmod>2024-08-09T09:39:27.628501+00:00</lastmod>
+    <changefreq>daily</changefreq>
+    <priority>0.5</priority>
+  </url>
```

Comment:
Can we include the subprojects as well, given that they aren't included in the sitemap https://github.com/readthedocs/readthedocs.org/issues/6841? hence

- https://docs.kedro.org/projects/kedro-viz/en/stable/
- https://docs.kedro.org/projects/kedro-viz/en/latest/
- https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-4.1.0/ (unfortunately there's no  `stable` tag for kedro-datasets, see https://github.com/readthedocs/readthedocs.org/issues/5319)
- https://docs.kedro.org/projects/kedro-datasets/en/latest/

Reply:
Actually I just implemented a `/stable` redirection for kedro-datasets

![image](https://github.com/user-attachments/assets/6c4b9153-e9a9-4c9b-9cf5-e7d0a4a39f08)

(more context https://github.com/readthedocs/readthedocs.org/issues/11183#issuecomment-2333405041)

So we can add

- https://docs.kedro.org/projects/kedro-datasets/en/stable/

Reply:
Many thanks @astrojuanlu , the redirection works well. I agree, projects added